server/service/src/requisition/response_requisition/create_requisition_shipment/generate.rs

```diff
@@ -94,6 +100,7 @@ pub fn generate_invoice_lines(
             item_code: item_row.code,
             item_name: item_row.name,
             r#type: InvoiceLineType::UnallocatedStock,
+            program_id: program_id.clone(),
```

Comment:
Apply the program id of the requisition (if it was a program requisition) to the generated invoice lines... therefore ending up as the program id on the stock lines when received

Reply:
Per change below - realised we shouldn't be setting program id here - outbound lines should have their program ID set based on the stock line they are allocated from :eyes: